assets/js/sections/gridmap.js

```diff
@@ -200,8 +200,8 @@ function spawnGridsquareModal(loc_4char) {
 					}]
 				});
 			    dialog.realize();
-		    		$("#gridsquare_map").append(dialog.getModal());
-		    		dialog.open();
+					$("#gridsquare_map").append(dialog.getModal());
```

Comment:
change the id instead will fix the issue
```suggestion
					$("body").append(dialog.getModal());
```